schemas/paconn-settings.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "https://github.com/microsoft/PowerPlatformConnectors/blob/master/schemas/paconn-settings.schema.json#",
+  "type": "object",
+  "required": [],
+  "additionalProperties": false,
+  "properties": {
+    "connectorId": {
+      "type": "string"
+    },
+    "environment": {
+      "type": "string"
+    },
+    "apiProperties": {
+      "type": "string",
+      "default": "apiProperties.json"
+    },
+    "apiDefinition": {
+      "type": "string",
+      "default": "apiDefinition.swagger.json"
+    },
+    "icon": {
+      "type": "string",
+      "default": "icon.png"
+    },
+    "powerAppsUrl": {
+      "type": "string",
+      "format": "uri",
+      "default": "https://api.powerapps.com",
```

Comment:
Can we add support for regional URLs (such as http://unitedstates.api.powerapps.com) as well? Do you think that will be helpful?